patches/tModLoader/Terraria/Localization/Content/en-US/tModLoader.json

```diff
@@ -338,6 +338,9 @@
 		// Misc
 		"OK": "OK",
 		"Continue": "Continue",
+		"Yes": "Yes",
```

Comment:
You should use `LegacyMenu.104` (Yes) and `LegacyMenu.105` (No). These are already localized.

Reply:
Did not know of those, thanks.